gradle/allprojects.gradle

```diff
@@ -25,5 +25,4 @@ allprojects {
   apply from: rootProject.file('gradle/common-dependencies.gradle')
   apply from: rootProject.file("gradle/lint.gradle")
   apply from: rootProject.file("gradle/tests.gradle")
-  apply from: rootProject.file("gradle/trace-files.gradle")
-}
\ No newline at end of file
+}
```

Comment:
The file `trace-files.gradle` was deleted, since most of it is redundant now.  The `buildTracer` task is now located in `corset.gradle`.